elastic-training/controllers/elastichorovodjob_controller.go

```diff
@@ -215,6 +320,15 @@ func (r *ElasticHorovodJobReconciler) desiredWorkers(ehjob aiv1alpha1.ElasticHor
 								},
 							},
 						},
+						//shared memory
```

Comment:
What is the use for this shared memory?

Reply:
PyTorch Horovod seems to require a shared memory mount